setup/user/user.go

```diff
@@ -0,0 +1,109 @@
+package user
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	cfg "github.com/codeready-toolchain/toolchain-e2e/setup/configuration"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	k8swait "k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Setup struct {
+	Config *rest.Config
+	Client client.Client
+}
+
+func (s Setup) UpdateUserIdlersTimeout(username string, timeout time.Duration) error {
+	for _, suffix := range []string{"code", "dev", "stage"} { // TODO: hard coded suffixes, we could probably get them from the tier instead
```

Comment:
I thought we said we would create a new tier with a shorter idler timeout? or did we decide to postpone it?

Reply:
yeah, that's still the plan. Just haven't go around to it yet.